vfs.go

```diff
@@ -29,6 +29,9 @@ type FileSystem interface {
 
 	// Scheme, related to Name, is the uri scheme used by the FileSystem: s3, file, gs, etc...
 	Scheme() string
+
+	// Retry will return the retry function to be used by any file system.
+	Retry() Retry
```

Comment:
Because I added this new function to the FileSystem interface, it will require us to do a full version bump of the library.